source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -557,6 +557,7 @@ message Transformation {
     TransformationWrapEarlyTerminatorInFunction wrap_early_terminator_in_function = 83;
     TransformationMergeFunctionReturns merge_function_returns = 84;
     TransformationExpandVectorReduction expand_vector_reduction = 85;
+    TransformationSwapFunctionVariables 
```

Comment:
Hmm, I doubt this would compile as you have not given the protobuf field a name nor a number.

Reply:
You also haven't defined this protobuf message - below there should be a definition of it with appropriate fields. Look at other transformation messages as examples.